### PR TITLE
Upload Balsamic SV research automatically

### PR DESCRIPTION
**Pull request by Mropat**
### Added

-

### Changed

- use vcf-sv-clinical tag when delivering balsamic wgs rawdata and uplaoding to scout

### How to prepare for test
- [ ] ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] paxa the environment: `paxa`
- [ ] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

**Comment by Mropat** [comment]
```
Fetch latest version from bundle lovedheron
Setting ticket_id to 863252
Setting customer_id to cust127
Hard link file /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/SV.somatic.lovedheron.manta.all.vcf.gz to /home/proj/stage/customers/cust127/inbox/863252/GMSB-ST004-2/SV.somatic.GMSB-ST004-2.manta.all.vcf.gz
Hard link file /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/SV.somatic.lovedheron.manta.all.vcf.gz.tbi to /home/proj/stage/customers/cust127/inbox/863252/GMSB-ST004-2/SV.somatic.GMSB-ST004-2.manta.all.vcf.gz.tbi
Hard link file /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/multiqc_report.html to /home/proj/stage/customers/cust127/inbox/863252/GMSB-ST004-2/multiqc_report.html
Hard link file /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/concatenated_ACC8983A1_XXXXXX_R.dedup.realign.cnr to /home/proj/stage/customers/cust127/inbox/863252/GMSB-ST004-2/concatenated_ACC8983A1_XXXXXX_R.dedup.realign.cnr
Hard link file /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/concatenated_ACC8983A1_XXXXXX_R.dedup.realign-diagram.pdf to /home/proj/stage/customers/cust127/inbox/863252/GMSB-ST004-2/concatenated_ACC8983A1_XXXXXX_R.dedup.realign-diagram.pdf
Hard link file /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/concatenated_ACC8983A1_XXXXXX_R.dedup.realign-scatter.pdf to /home/proj/stage/customers/cust127/inbox/863252/GMSB-ST004-2/concatenated_ACC8983A1_XXXXXX_R.dedup.realign-scatter.pdf
Hard link file /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/SNV.somatic.lovedheron.tnscope.filtered.pass.vcf.gz to /home/proj/stage/customers/cust127/inbox/863252/GMSB-ST004-2/SNV.somatic.GMSB-ST004-2.tnscope.filtered.pass.vcf.gz
Hard link file /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/SNV.somatic.lovedheron.tnscope.filtered.pass.vcf.gz.tbi to /home/proj/stage/customers/cust127/inbox/863252/GMSB-ST004-2/SNV.somatic.GMSB-ST004-2.tnscope.filtered.pass.vcf.gz.tbi
Linked 8 files for case lovedheron
Hard link file /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/normal.merged.cram to /home/proj/stage/customers/cust127/inbox/863252/GMSB-ST004-2/MOL1545-21G/normal.merged.cram
Hard link file /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/normal.merged.cram.crai to /home/proj/stage/customers/cust127/inbox/863252/GMSB-ST004-2/MOL1545-21G/normal.merged.cram.crai
Linked 2 files for sample ACC8237A4, case lovedheron
Hard link file /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/tumor.merged.cram to /home/proj/stage/customers/cust127/inbox/863252/GMSB-ST004-2/MOL3000-21T/tumor.merged.cram
Hard link file /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/tumor.merged.cram.crai to /home/proj/stage/customers/cust127/inbox/863252/GMSB-ST004-2/MOL3000-21T/tumor.merged.cram.crai
Linked 2 files for sample ACC8983A1, case lovedheron

```

**Comment by Mropat** [comment]
```
Set root dir to /home/proj/stage/cancer/cases
analysis_date: 2021-12-08 02:01:14
default_gene_panels: []
family: lovedheron
family_name: GMSB-ST004-2
human_genome_build: '37'
multiqc: /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/multiqc_report.html
owner: cust127
rank_score_threshold: -100
samples:
- alignment_path: /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/normal.merged.cram
  analysis_type: wgs
  phenotype: unaffected
  sample_id: NORMAL
  sample_name: MOL1545-21G
  sex: female
  tissue_type: blood
  tumor_purity: 0
- alignment_path: /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/tumor.merged.cram
  analysis_type: wgs
  phenotype: affected
  sample_id: TUMOR
  sample_name: MOL3000-21T
  sex: female
  subject_id: ''
  tissue_type: unknown
  tumor_purity: 0
synopsis: ''
track: cancer
vcf_cancer: /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/SNV.somatic.lovedheron.tnscope.filtered.pass.vcf.gz
vcf_cancer_sv: /home/proj/stage/housekeeper-bundles/lovedheron/2021-12-08/SV.somatic.lovedheron.manta.all.vcf.gz

Would save file to /home/proj/stage/cancer/cases/lovedheron/scout_load.yaml
```

**Comment by Mropat** [comment]
@Clinical-Genomics/bioinfo @henningonsbring @karlnyr 
Does this solve balsamic upload automation?

File: cg/constants/scout_upload.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 BALSAMIC_CASE_TAGS = dict(
     snv_vcf={"vcf-snv-clinical"},
-    sv_vcf={"vcf-sv-clinical"},
```
Comment (henningonsbring) [inline cg/constants/scout_upload.py:17]: do we have anything that is under this tag that wont be delievered now?

**Comment by henningonsbring** [comment]
This PR should be tested by running `cg deliver analysis -d balsamic -c <petname>` to check that we get the files we want into the customer inbox on hasta. The customer inbox is what we finally rsync to the customer.

File: cg/constants/scout_upload.py (hunk 1 above)
Comment (Mropat) [reply cg/constants/scout_upload.py:17]: Nope, there never was anything under the flag.

Comment (moahaegglund) [reply cg/constants/scout_upload.py:17]: It's back in the new version of balsamic, it's the filtered vcf.

**Review by henningonsbring — APPROVED** [review]

**Comment by Mropat** [comment]
Being completed by @ivadym in other PR

**Pull request closed without merging**